support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -366,18 +450,19 @@ export function Checkout() {
 		const ophanIds = getOphanIds();
 		const referrerAcquisitionData = getReferrerAcquisitionData();
 
-		if (paymentFields) {
+		if (paymentFields && productFields) {
 			/** TODO - Remove this Omit to make the data valid to the endpoint */
 			const createSupportWorkersRequest: Omit<
 				RegularPaymentRequest,
-				'product' | 'firstDeliveryDate' | 'supportAbTests' | 'debugInfo'
+				'firstDeliveryDate' | 'supportAbTests' | 'debugInfo'
 			> = {
 				...personalData,
 				billingAddress,
 				deliveryAddress,
 				paymentFields,
 				ophanIds,
 				referrerAcquisitionData,
+				product: productFields,
```

Comment:
Is `createSupportWorkersRequest` currently not called du eto the omit and will be modified presume to take the new `productFields`?

Reply:
[It is called a couple lines below](https://github.com/guardian/support-frontend/pull/5930/files#diff-dbc5d74a5f52e0cc93a8185ea2825653897711f50a6af3029dd91a90bcdd145fR467).

It won't validate though.